src/xenia/kernel/xam/xam_nui.cc

```diff
@@ -35,23 +57,298 @@ struct X_NUI_DEVICE_STATUS {
+
+dword_result_t XamUserNuiGetUserIndexForBind_entry(lpdword_t index) {
+  return X_E_FAIL;
+}
+DECLARE_XAM_EXPORT1(XamUserNuiGetUserIndexForBind, kNone, kStub);
+
+dword_result_t XamNuiGetDepthCalibration_entry(lpdword_t unk1) {
+  /* Notes:
+     - Possible returns X_STATUS_NO_SUCH_FILE, and 0x10000000
+  */
+  return X_STATUS_NO_SUCH_FILE;
+}
+DECLARE_XAM_EXPORT1(XamNuiGetDepthCalibration, kNone, kStub);
+
+// Skeleton
+qword_result_t XamNuiSkeletonGetBestSkeletonIndex_entry(int_t unk) {
+  return 0xffffffffffffffff;
+}
+DECLARE_XAM_EXPORT1(XamNuiSkeletonGetBestSkeletonIndex, kNone, kStub);
+
+/* XamNuiCamera Notes
+   - most require message calls to xam in 0x0002Bxxx area
+*/
+
+dword_result_t XamNuiCameraTiltGetStatus_entry(lpvoid_t unk) {
+  /* Notes:
+     - Used by XamNuiCameraElevationGetAngle, and XamNuiCameraSetFlags
+     - if it returns anything greater than -1 then both above functions continue
+     - Both funcs send in a param of *unk = 0x50 bytes to copy
+     - unk2
+     - Ghidra decompile fails
+  */
+  return X_E_FAIL;
+}
+DECLARE_XAM_EXPORT1(XamNuiCameraTiltGetStatus, kNone, kStub);
+
+dword_result_t XamNuiCameraElevationGetAngle_entry(lpqword_t unk1,
+                                                   lpdword_t unk2) {
+  /* Notes:
+     - Xam 12611 does not show what unk1 is used for (Ghidra)
+  */
+  uint32_t tilt_status[] = {0x58745373, 0x50};  // (XtSs)? & bytes to copy
+  uint32_t response;
+  response = XamNuiCameraTiltGetStatus_entry(tilt_status);
+  if (response > -1) {
+    // operation here
+    // *unk1 = output1
+    // *unk2 = output2
+  }
+  return response;
+}
+DECLARE_XAM_EXPORT1(XamNuiCameraElevationGetAngle, kNone, kStub);
+
+dword_result_t XamNuiCameraGetTiltControllerType_entry() {
+  /* Notes:
+     - undefined unk[8]
+     - undefined8 local_28;
+     - undefined8 local_20;
+     - undefined8 local_18;
+     - undefined4 local_10;
+     - local_20 = 0;
+     - local_18 = 0;
+     - local_10 = 0;
+     - local_28 = 0xf030000000000;
+     - calls DetroitDeviceRequest(unk) -> result
+     - returns (ulonglong)(LZCOUNT(result) << 0x20) >> 0x25
+  */
+  return X_E_FAIL;
+}
+DECLARE_XAM_EXPORT1(XamNuiCameraGetTiltControllerType, kNone, kStub);
+
+dword_result_t XamNuiCameraSetFlags_entry(qword_t unk1, dword_t unk2) {
+  /* Notes:
+     - if XamNuiCameraGetTiltControllerType returns 1 then operation is done
+     - else 0xffffffff8007048f
+  */
+  uint32_t result = X_E_DEVICE_NOT_CONNECTED;
+  int Controller_Type = XamNuiCameraGetTiltControllerType_entry();
+
+  if (Controller_Type == 1) {
+    uint32_t tilt_status[] = {0x58745373, 0x50};  // (XtSs)? & bytes to copy
+    result = XamNuiCameraTiltGetStatus_entry(tilt_status);
+    if (result > -1) {
+      // op here
+      // result =
+    }
+  }
+  return result;
+}
+DECLARE_XAM_EXPORT1(XamNuiCameraSetFlags, kNone, kStub);
+
+// Is
+dword_result_t XamIsNuiUIActive_entry() { return xam_dialogs_shown_ > 0; }
```

Comment:
I'm concerned about this part. I wouldn't assume that every xam dialog is Nui type UI